pkg/kubelet/types/constants.go

```diff
@@ -38,3 +38,16 @@ const (
 	LimitedSwap SwapBehavior = "LimitedSwap"
 	NoSwap      SwapBehavior = "NoSwap"
 )
+
+// Pod status condition reasons and messages
+const (
+	// PodReadyToStartContainers condition reason
+	PodSandboxNotReadyReason = "PodSandboxNotReady"
+
+	// PodReadyToStartContainers condition messages
+	PodSandboxNotReadyMsgNoPodSandbox         = "no pod sandbox exists"
+	PodSandboxNotReadyMsgMultipleSandboxes    = "multiple pod sandboxes are ready"
```

Comment:
missing two of these in the tests, can there be a comment at least saying TODO to catch them, or explain why it's not possible?

Reply:
Added the test for `PodSandboxNotReadyMsgMultipleSandboxes`